### DISCUSSION TITLE
Comprehension question: Multiple API servers with repeatable jobs

### DISCUSSION THREAD

**Discussion by trnc-ck** [taskforcesh/bullmq#1293] [General]
hey,

i have a bit of an understanding problem. Running one server instance which launches one queue and one worker, works like a charm. Now, what if the api is behind a load balancer and the api launches multiple queues and workers (as it's the same code as a single instance) and we have repeatable jobs, we run every 10sec. Does every instance run the same job every 10sec and the queue is messed up?

thanks!

**Comment by manast** [comment]
Nah, it works. If it messed things up there would be no point in a library like this, right? :)

**Reply by trnc-ck** [reply]
well, used other libraries in python which actually executed scheduled repeating tasks twice then…

thanks for the info.
